60.md

```diff
@@ -0,0 +1,188 @@
+# NIP-60
+## Cashu Wallet
+`draft` `optional`
+
+This NIP defines the operations of a cashu-based wallet.
+
+A cashu wallet is a wallet which information is stored in relays to make it accessible across applications.
+
+The purpose of this NIP is:
+* ease-of-use: new users immediately are able to receive funds without creating accounts with other services.
+* interoperability: users' wallets follows them across applications.
+
+This NIP doesn't deal with users' *receiving* money from someone else, it's just to keep state of the user's wallet.
+
+# High-level flow
+1. A user has a `kind:37375` event that represents a wallet.
+2. A user has `kind:7375` events that represent the unspent proofs of the wallet. -- The proofs are encrypted with the user's private key.
```

Comment:
you can encrypt (NIP-44) with your nostr private key

Reply:
No you don't: For NIP-44, you calculate a shared secret out of sender's private key + recipient's public key, then you calculate a message key out of that, then you encrypt with that. Saying "you can encrypt (NIP-44) with your nostr private key" is confusing. You need to state the sender and recipient for NIP-44.

My suggestion would be to simply write "the proofs are encrypted with NIP-44" if the above is what you mean. Encrypting with a private key is an oxymoron.

However, maybe the confusion arises from this: I can't tell who the recipient is since there's never any `p` tag in this spec (by design) from which the recipient's public key could be deduced. Therefore, I don't know which sk+pk I should use for NIP-44 for the wallet and token event. Does the following mean you're using sk+pk of the same key pair since you're essentially sending these events to yourself according to `<my-pubkey>`:

> **Fetch wallet and token list**
>
> From those relays, the client should fetch wallet and token events.
>
> `"kinds": [37375, 7375], "authors": ["<my-pubkey>"]`

? (not sure if that's even possible)